.github/workflows/extension.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.11"]
+        python-version: ["3.9", "3.11", "3.12"]
```

Comment:
Little confused here, why we just skip `3.10` version here?  See `python-package` contained `3.10` though.

Reply:
My intention was to wait until python 3.12 was stable to test with just 3.9 and 3.12, and so far 3.12 seems to be stable enough. Switching to 3.12

Reply:
Yeah. I might not have been very clear. Here I am actually confused about the `matrix`.

As you assigned it  ` python-version: ["3.9", "3.12"]` in `ci-test-extension.yml` , but why its matirx value `["3.9", "3.10", "3.11", "3.12"]` in `ci-test-python-package.yml`?

Should we assigned `["3.9", "3.10", "3.11", "3.12"]` value in `ci-test-extension.yml`?

Reply:
There's no need for that, for the plugin there's no computational library dependency so there's basically no python version difference, we just need to make sure it runs on the oldest and newest (supproted) versions. For the core library, I'd like to make sure he runs on all supported python versions.